src/plugins/viewport.ts

```diff
@@ -1,4 +1,5 @@
-import { IPlugin, IViewportState } from "../../types/index";
+import { IPlugin } from "../../types/core";
+import { IViewportState } from "../../types/viewport";
```

Comment:
>./.. [](start = 33, length = 4)

Could you path aliases to make these path saner?

Reply:
Not yet, some infrastructure changes from other PRs are required to use path aliases

Reply:
Not yet, some infrastructure changes from other PRs are required to use path aliases

---
In reply to: [250244043](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250244043) [](ancestors = 250244043)